ui-specs/README.md

```diff
@@ -94,6 +110,19 @@ The primary objective of this document is to develop a flutter dating app that a
    - LocationPermissionGranted: State when permission is Granted.
    - LocationPermissionDenied: State when Permission is Denied.
 
+2. ### AccountSetupPage1Bloc
+
+   -Events
+
+   - EventName: Event to capture Profile Name
```

Comment:
Property:
- CreateProfileModel: store profile details provided by user.
- Map<String, String> fieldErrors: contains field name and error message. for e.g incase of any invalid input the backend will return error message `{'field_name': ['error reason 1', 'error reson 2'], 'field_name_2': ['error reason 3']}`. Create a map within the bloc to for mapping fields with pages (which page contains what field) and show the error message on the input field
Events:
- UpdateCreateProfileModelCreateProfileEvent(CreateProfileModel profile): Update the CreateProfileModel propterty in the bloc.
- SaveCreateProfileModelEvent(): call create profile api and save the profile

States:
- InitialiCreateProfileState
- PreviewCreateProfileFormsCreateProfileState(CreateProfile? profile, Map<String, String>? fieldErrors): Pages will use BlocBuilder to pull CreateProfileBloc and show the respective page. The page can use the `profile` for prefilling inputs.
- SavingCreateProfileModelCreateProfileState: show indicator like loading indicator while making api request
- SavedCreateProfileModelCreateProfileState: redirect back to initialization activity
- SaveCreateProfileModelFailedCreateProfileState: Profile creation failed due to some error. Django returns field name that contains error, in case any invalid value was provided, you can implement an intelligent redirector which will redirect to the page which contains error and show the error message. Otherwise you can navigate to first page and show error snack bar of unknow error.

Reply:
Use field names similar to backend for easy mapping. Backend follows snake case.